CONTRIBUTING.md

```diff
@@ -90,6 +90,40 @@ If you want the team to review the changes, mark the pull request as "Ready for
 
 After the pull request has been reviewed and approved, the feature branch will be merged into the main branch and automatically closed and deleted.
 
+## Steering Committee
+
+The Enterprise-Scale Analytics Committee and its members (aka Committee Members) are the primary caretakers of the Enterprise-Scale Analytics repos including language, design, policy oversight, and reference implementations.
+
+Voting members may be added once approved by the Steering Committee. All members abide by all Organization polices, including the code of conduct. Voting members may be added or removed by no less than 3/4 affirmative vote of the Steering Committee. The membership of a Steering Committee member can terminate in case of written notice of resignation or if the member is unreachable or unresponsive for more than three months.
```

Comment:
"Organization polices" change to "organizational policies"

Reply:
Changed